go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -31,7 +31,7 @@ func (h *headerArgs) Set(value string) error {
 
 var (
 	flagUri            = flag.String("uri", "https://example.com/index.html", "The URI of the resource represented in the exchange")
-	flagVersion        = flag.String("version", "1b1", "The signedexchange version")
+	flagVersion        = flag.String("version", "1b2", "The signedexchange version")
```

Comment:
Intended? (If so, please note in the patch description).

Reply:
no. Thanks for the catch